### PR TITLE
voting hearts 1

### PR DESCRIPTION
**Pull request by Ramotka**

### CODE REVIEW

File: projects/feature/src/lib/adapters/primary/components/feature-list.component.html [hunk 1]
```diff
@@ -4,7 +4,10 @@
     class="flex items-start bg-indigo-50 w-[50rem] p-5 rounded-md my-3"
   >
     <div class="flex justify-center items-center w-1/12 mt-2">
-      <div (click)="onVotingIconClicked(feature)">
+      <div
+        (click)="onVotingIconClicked(feature)"
```
Comment (gradzio) [inline projects/feature/src/lib/adapters/primary/components/feature-list.component.html:8]: this is weird, we should always have 1 click handler per element

File: projects/feature/src/lib/adapters/primary/components/feature-list.component.ts [hunk 2]
```diff
@@ -42,10 +45,29 @@ import {
...
+    .select()
+    .pipe(
+      switchMap((selectedFeatureId) =>
+        combineLatest([
+          this._getsCurrentLoggedUserQuery.getCurrentLoggedUserQuery(),
+          this._getsOneFeatureDto.getOne(selectedFeatureId as string),
+        ]).pipe(
+          take(1),
+          switchMap(([currentUserId, feature]) => {
+            if (feature.voters.includes(currentUserId)) {
```
Comment (gradzio) [inline projects/feature/src/lib/adapters/primary/components/feature-list.component.ts:61]: this should be in a query. Please make FeatureQuery that will contain all these info

File: projects/feature/src/lib/adapters/primary/components/feature-list.component.html (hunk 1 above)
Comment (Ramotka) [reply projects/feature/src/lib/adapters/primary/components/feature-list.component.html:8]: ok removed

File: projects/feature/src/lib/adapters/primary/components/feature-list.component.ts (hunk 2 above)
Comment (Ramotka) [reply projects/feature/src/lib/adapters/primary/components/feature-list.component.ts:61]: ok done

File: projects/shared/tsconfig.lib.json [hunk 3]
```diff
@@ -7,16 +7,12 @@
...
   },
-  "exclude": [
-    "src/test-setup.ts",
-    "**/*.spec.ts"
-  ],
+  "exclude": ["src/test-setup.ts", "**/*.spec.ts"],
   "include": [
-    "**/*.ts"
+    "**/*.ts",
+    "../feature/src/lib/application/ports/primary/query/logged-user.query.ts",
```
Comment (gradzio) [inline projects/shared/tsconfig.lib.json:15]: it should not be here

File: projects/shared/tsconfig.lib.json [hunk 4]
```diff
@@ -7,16 +7,12 @@
...
+    "../feature/src/lib/application/ports/primary/query/gets-current-logged-user.query-port.ts"
```
Comment (gradzio) [inline projects/shared/tsconfig.lib.json:16]: it should not be here

File: projects/feature/tsconfig.lib.json [hunk 5]
```diff
@@ -7,16 +7,11 @@
...
   },
-  "exclude": [
-    "src/test-setup.ts",
-    "**/*.spec.ts"
-  ],
+  "exclude": ["src/test-setup.ts", "**/*.spec.ts"],
   "include": [
-    "**/*.ts"
+    "**/*.ts",
+    "../shared/src/lib/application/ports/primary/query/gets-current-logged-user.query-port.ts"
```
Comment (gradzio) [inline projects/feature/tsconfig.lib.json:15]: it should not be here

**Pull request closed without merging**